legacy/src/api/merchants/merchants.md

```diff
@@ -296,6 +296,8 @@ Returns a [paginated][] list of Merchants attached to an Account.
 
 Returns a [paginated][] response of [merchant search results](#merchant-search-result) that match the search query
 
+This endpoint does not require auth
```

Comment:
@bpal938 Capitalise Auth and link it to the Auth documentation (https://docs.centrapay.com/api/auth)

Reply:
Good idea, i've added this now